include/openssl/conf_api.h

```diff
@@ -10,31 +10,8 @@
 #ifndef  HEADER_CONF_API_H
 # define HEADER_CONF_API_H
 
-# include <openssl/lhash.h>
-# include <openssl/conf.h>
+# ifndef  HEADER_CONF_H
+#  include <openssl/conf.h>
+# endif
```

Comment:
Why is the guard necessary?

Reply:
I was worried about mutually-recursive header includes.  Maybe don't bother?

Reply:
I removed the guard.